package.json

```diff
@@ -24,46 +24,46 @@
     "@angular/platform-browser-dynamic": "^19.0.5",
     "@angular/router": "^19.0.5",
     "@ngrx/store": "^19.0.0",
-    "deepmerge": "^4.2.2",
+    "deepmerge": "^4.3.1",
     "ngrx-store-localstorage": "file:projects/lib",
-    "rxjs": "~7.8.1",
-    "tslib": "^2.6.2",
-    "zone.js": "~0.15.0"
+    "rxjs": "~7.8.2",
+    "tslib": "^2.8.1",
+    "zone.js": "~0.15.1"
   },
   "devDependencies": {
     "@angular-devkit/build-angular": "^19.0.6",
-    "@angular-eslint/builder": "19.0.2",
-    "@angular-eslint/eslint-plugin": "19.0.2",
-    "@angular-eslint/eslint-plugin-template": "19.0.2",
-    "@angular-eslint/schematics": "19.0.2",
-    "@angular-eslint/template-parser": "19.0.2",
+    "@angular-eslint/builder": "19.7.1",
+    "@angular-eslint/eslint-plugin": "19.7.1",
+    "@angular-eslint/eslint-plugin-template": "19.7.1",
+    "@angular-eslint/schematics": "19.7.1",
+    "@angular-eslint/template-parser": "19.7.1",
     "@angular/cli": "^19.0.6",
     "@angular/compiler-cli": "^19.0.5",
-    "@commitlint/cli": "^12.1.4",
-    "@commitlint/config-conventional": "^12.1.4",
-    "@istanbuljs/nyc-config-typescript": "^1.0.1",
-    "@types/jasmine": "~3.6.0",
+    "@commitlint/cli": "^19.8.1",
+    "@commitlint/config-conventional": "^19.8.1",
+    "@istanbuljs/nyc-config-typescript": "^1.0.2",
+    "@types/jasmine": "~5.1.8",
     "@types/node": "^18.16.0",
-    "@typescript-eslint/eslint-plugin": "^5.59.2",
-    "@typescript-eslint/parser": "^5.59.2",
+    "@typescript-eslint/eslint-plugin": "^8.33.1",
+    "@typescript-eslint/parser": "^8.33.1",
     "copyfiles": "^2.4.1",
-    "crypto-js": "^4.0.0",
-    "es6-shim": "^0.35.6",
-    "eslint": "^8.39.0",
-    "eslint-config-prettier": "^10.1.2",
-    "husky": "^6.0.0",
-    "jasmine": "^3.10.0",
-    "jasmine-core": "~3.10.0",
+    "crypto-js": "^4.2.0",
+    "es6-shim": "^0.35.8",
+    "eslint": "^9.28.0",
+    "eslint-config-prettier": "^10.1.5",
+    "husky": "^9.1.7",
+    "jasmine": "^5.7.1",
+    "jasmine-core": "~5.7.1",
     "jasmine-spec-reporter": "^7.0.0",
-    "lint-staged": "^11.0.0",
+    "lint-staged": "^16.1.0",
     "localstorage-polyfill": "^1.0.1",
     "ng-packagr": "^19.0.1",
-    "nyc": "^15.1.0",
+    "nyc": "^17.1.0",
     "prettier": "^3.5.3",
-    "rimraf": "^3.0.2",
-    "semantic-release": "^17.4.4",
-    "ts-node": "^10.8.1",
-    "typescript": "^5.6.3"
+    "rimraf": "^6.0.1",
+    "semantic-release": "^24.2.5",
```

Comment:
Several of these are major bumps. 

Have you had a look to see if this project is impacted by any of the breaking changes?

Reply:
For example. Semantic release v17 --> v24